tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/util/ProjectImportUtil.java

```diff
@@ -42,6 +43,10 @@
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.io.filefilter.DirectoryFileFilter;
 import org.apache.commons.io.filefilter.WildcardFileFilter;
+import org.eclipse.buildship.core.projectimport.ProjectImportConfiguration;
```

Comment:
Right now yes, but eventually we will have non-gradle modules like maven and bndtools so project.core must be gradle agnostic.